source/authentication/custom-function.txt

```diff
@@ -79,75 +79,18 @@ included with the custom function provider credential in the client app.
 The function accepts any value provided from your client app, so the
 actual field names and values depend on your implementation.
 
-.. example::
+For examples of how to create a custom credential ``payload`` object
+using the Realm SDKs, refer to the documentation for a specific SDK:
 
-   The following code examples show how to create a function credential
-   using the Realm SDKs. The credential includes a ``payload`` object
-   that is passed to the authentication function.
-
-   .. tabs-realm-sdks::
-   
-      .. tab::
-         :tabid: javascript
-   
-         .. code-block:: javascript
-         
-            const credentials = Realm.Credentials.function({
-              username: "someUsername",
-              pin: "24601",
-            });
-   
-      .. tab::
-         :tabid: android
-   
-         .. code-block:: java
-            
-            import org.bson.Document;
-            import static java.util.Map.entry;
-   
-            val credentials: Credentials = Credentials.customFunction(
-              Document(
-                Map.ofEntries(
-                  entry("username", "someUsername"),
-                  entry("pin", "24601")
-                )
-              )
-            )
-   
-      .. tab::
-         :tabid: dotnet
-   
-         .. code-block:: csharp
-         
-            var credentials = Credentials.Function(new {
-              username = "someUsername",
-              pin = "24601"
-            })
-   
-      .. tab::
-         :tabid: ios
-   
-         .. code-block:: swift
-         
-            let credentials = Credentials.function(payload: [
-               "username": "someUsername",
-               "pin": "24601"
-            ])
-
-      .. tab::
-         :tabid: flutter
-
-         .. code-block:: dart
-
-            Map<String, String> credentials = {
-               "username": "someUsername",
-               "pin": "24601"
-            };
-            // payload must be a JSON-encoded string
-            String payload = jsonEncode(credentials);
-
-            Credentials customCredentials = Credentials.function(payload);
-            User currentUser = await app.logIn(customCredentials);
+- :ref:`C++ SDK (Alpha) <cpp-login-custom-function>`
```

Comment:
```suggestion
- :ref:`C++ SDK <cpp-login-custom-function>`
```

nit more maintainable to not include "Alpha" here for when it's updated to beta, rc, ga, etc

Reply:
Good catch, thanks!